src/Lykke.Service.BlockchainWallets.Core/Exceptions/ErrorType.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Lykke.Service.BlockchainWallets.Core.Exceptions
+{
+    public enum ErrorType
```

Comment:
To distinguish domain and contract error codes, I suggest to name domain one ```OperationErrorCode``` since it's used in ```OperationException```